### PR TITLE
Neon Effect Filter #1117

### LINKED ISSUE #1117: Neon Effect Filter
**Is your feature request related to a problem? Please describe.**
Want to apply a "Neon Effect Filter" on an image.

**Describe the solution you'd like**
Implementing a neon effect filter using OpenCV, Matplotlib, Os and NumPy.

**Final Output**
Screenshot (396)

### PR DESCRIPTION
**Pull request by A-kriti**
# Description

Added new folder "Neon Effect Filter" containing all the required files!

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows Template for README.md
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

## Project Metadata

Category:
- [ ] Calculators
- [ ] AI/ML
- [ ] Scrappers
- [ ] Social_Media
- [ ] PDF
- [x] Image_Processing
- [ ] Video_Processing
- [ ] Games
- [ ] Networking
- [ ] OS_Utilities
- [ ] Automation
- [ ] Cryptography
- [ ] Computer_Vision
- [ ] Fun
- [ ] Others

Title: Neon Effect Filter

Folder: Neon Effect Filter

Requirements: requirements.txt

Script: Code.py

Arguments: none

Contributor: Akriti

Description: Implementing a "Neon Effect Filter" using OpenCV, Matplotlib, Os, and NumPy.

### CODE REVIEW

File: Neon Effect Filter/Code.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+img_edge = cv2.cvtColor(img_edge, cv2.COLOR_GRAY2RGB)
+array = cv2.bitwise_and(image, img_edge)
+array1 = cv2.bitwise_xor(array, image)
+plt.figure(figsize= (10,10))
+plt.imshow(array1,cmap='gray')
+plt.title("Neon Effect Filtered Photo")
+plt.axis('off')
+filename = os.path.basename(image_path)
+plt.savefig("(Neon Effect Filter)"+filename)  #saved file name as (Filtered)image_name.jpg
```
Comment (kaustubhgupta) [inline Neon Effect Filter/Code.py:29]:
```suggestion
plt.savefig("./Neon Effect Filter/Neon Effect Filter"+filename)  #saved file name as (Filtered)image_name.jpg
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
- PR template wrongly updated
- Other changes:

**Review by kaustubhgupta — APPROVED** [review]
cool 👍🏼 💯

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..🚩

**Pull request merged**